apps/kitchen-sink/src/ensemble/screens/actions.yaml

```diff
@@ -192,6 +192,10 @@ View:
                                 styles:
                                   fontSize: ${typography.fontSize['14']}
                                   fontWeight: ${typography.fontWeight.light}
+                      onDialogDismiss:
```

Comment:
This should be onModalDismiss right?

Reply:
Yes, you are right, I renamed after test, and forgot to update the YAML